nixos/modules/services/backup/restic.nix

```
@@ -292,7 +301,14 @@ in
         (name: backup:
           let
             extraOptions = concatMapStrings (arg: " -o ${arg}") backup.extraOptions;
-            resticCmd = "${backup.package}/bin/restic${extraOptions}";
+            inhibitCmd = concatStringsSep " " [
+              "${pkgs.systemd}/bin/systemd-inhibit"
+              "--mode='block'"
+              "--who='restic'"
+              "--what='sleep'"
+              "--why='Scheduled backup ${name}' "
+            ];
```

Why split the string beforhand?
```suggestion
            inhibitCmd = "${pkgs.systemd}/bin/systemd-inhibit --mode='block' --who='restic' --what='sleep' --why='Scheduled backup ${name}' "
            ];
```

---

I split it up for readability, no other reason. I could change it, but the next reviewer would probably tell me to split it up as it is too long:D